app/models/statistics/average_auction_length.rb

```diff
@@ -16,6 +16,6 @@ def total_time_length
   end
 
   def published_auctions
-    @_published_auctions ||= Auction.published
+    @_published_auctions ||= Auction.not_archived.published
```

Comment:
we should make sure that all of the statistics are not looking at archived auctions